docs/card/index.html

```diff
@@ -0,0 +1,203 @@
+          style="width: 680px; height: 392px; border:0; transform: scale(1); overflow:hidden;"
+          sandbox="allow-scripts allow-same-origin">
+        </iframe>
+
+        <h2 class="cl-sub-heading">Cards with Shadow</h2>
+        <p class="cl-text">For Shadow around a card.</p>
+        <div class="showcase showcase-col" >
+
+            <!-- Card with Shadow -->
+            <div class="card card-shadow">
+              <img class="card-image" src="https://picsum.photos/200/150" alt="" />
+              <p class="card-title">Our changing planet</p>
+              <p class="card-subtitle">By Kurl Wagner</p>
+              <p class="card-description">
+                Visit ten places on our planet that are under going the biggest
+                changes today.
+              </p>
+            </div>
+          </div>
+        <iframe
+          src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=material&wt=none&l=htmlmixed&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522card%2520card-shadow%2522%253E%250A%2520%2520%253Cimg%2520class%253D%2522card-image%2522%2520src%253D%2522https%253A%252F%252Fpicsum.photos%252F200%252F150%2522%2520alt%253D%2522%2522%2520%252F%253E%250A%2520%2520%253Cp%2520class%253D%2522card-title%2522%253EOur%2520changing%2520planet%253C%252Fp%253E%250A%2520%2520%253Cp%2520class%253D%2522card-subtitle%2522%253EBy%2520Kurl%2520Wagner%253C%252Fp%253E%250A%2520%2520%253Cp%2520class%253D%2522card-description%2522%253E%250A%2520%2520%2520%2520Visit%2520ten%2520places%2520on%2520our%2520planet%2520that%2520are%2520under%2520going%2520the%2520biggest%250A%2520%2520%2520%2520changes%2520today.%250A%2520%2520%253C%252Fp%253E%250A%253C%252Fdiv%253E"
+          style="width: 680px; height: 374px; border:0; transform: scale(1); overflow:hidden;"
+          sandbox="allow-scripts allow-same-origin">
+        </iframe>
+
+          <h2 class="cl-sub-heading">Vertical, Horizontal Cards</h2>
+        <p class="cl-text">Card layout can also be made Vertical and Horizontal.</p>
+        <div class="showcase showcase-col" >
+
+            <!-- Vertical Card -->
+            <div class="card">
+              <img class="card-image" src="https://picsum.photos/200/150" alt="" />
+              <p class="card-title">Our changing planet</p>
+              <p class="card-subtitle">By Kurl Wagner</p>
+              <p class="card-description">
+                Visit ten places on our planet that are under going the biggest
+                changes today.
+              </p>
+            </div>
+
+            <!-- Horizontal Card -->
+            <div class="card-horizontal">
+              <img class="card-image-horizontal" src="https://picsum.photos/200/150" alt="" />
```

Comment:
Please provide a meaningful alt description This is present in several other places.

Reply:
ok got it,